dev-util/clion/clion-2017.3.4.ebuild

```diff
@@ -10,7 +10,10 @@ HOMEPAGE="http://www.jetbrains.com/clion"
 SRC_URI="http://download.jetbrains.com/cpp/CLion-${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="IDEA
```

Comment:
Do you really need to unconditionally agree to commercial license when you want to use one of the licenses below?

Reply:
I don't know. I couldn't find a license for CLion. Neither on JetBrains website nor in the downloaded tar.gz from CLion. All licenses shipped with the tar.gz are related to components used in or bundled with CLion.

But I've found out that CLion doesn't has a community version that is free to use for open source development, like IDEA has. Thats the reason I've removed the IDEA_OpenSource license.

At work I've got a license key from JetBrains to use CLion, but no file with the license text.